Mathlib/Data/Finset/Slice.lean

```diff
@@ -120,21 +120,21 @@ def slice (𝒜 : Finset (Finset α)) (r : ℕ) : Finset (Finset α) := {A ∈ 
 scoped[Finset] infixl:90 " # " => Finset.slice
 
 /-- `A` is in the `r`-th slice of `𝒜` iff it's in `𝒜` and has cardinality `r`. -/
-theorem mem_slice : A ∈ 𝒜 # r ↔ A ∈ 𝒜 ∧ #A = r :=
+theorem mem_slice : A ∈ 𝒜 #r ↔ A ∈ 𝒜 ∧ #A = r :=
```

Comment:
This is a linter false positive: please revert this change, throughout this file. This needs a pretty-printer or linter fix.

Reply:
Reverted